packages/node/test/unit/channels.spec.ts

```diff
@@ -0,0 +1,48 @@
+import * as cf from "@counterfactual/cf.js";
+import { ethers } from "ethers";
+import * as _ from "lodash";
+
+import { Node, StateChannelInfoImpl } from "../../src/node";
+
+describe("Node can contain channels", () => {
+  it("Node has no channels", () => {
+    const channels = {};
+    const node = new Node(channels, cf.legacy.network.EMPTY_NETWORK_CONTEXT);
+    expect(node.channelStates).toEqual({});
+  });
+
+  it("Node has only 1 channel", () => {
+    const channels = {
+      [ethers.Wallet.createRandom().address]: new StateChannelInfoImpl(
+        ethers.constants.AddressZero,
+        ethers.constants.AddressZero,
+        ethers.constants.AddressZero,
+        {},
+        cf.legacy.utils.EMPTY_FREE_BALANCE
+      )
+    };
+    const node = new Node(channels, cf.legacy.network.EMPTY_NETWORK_CONTEXT);
+    expect(_.size(node.channelStates)).toEqual(1);
+  });
+
+  it("Node has multiple channels", () => {
+    const channels = {
+      [ethers.Wallet.createRandom().address]: new StateChannelInfoImpl(
```

Comment:
`ethers.Wallet.createRandom().address` => `ethers.utils.hexlify(ethers.utils.randomBytes(20))`

However as @mvanderh mentioned the other day, it's better to not literally compute random values each test.

Reply:
I recall that as well. Generally 👍 to not compute random values each time.